docs/dev/build_linux.md

```diff
@@ -74,9 +74,9 @@ You can use the following additional build options:
      ```
   2. Enable the `-DENABLE_PYTHON=ON` option in the CMake step above (Step 4). To specify an exact Python version, use the following options:
      ```
-     -DPYTHON_EXECUTABLE=`which python3.7` \
-     -DPYTHON_LIBRARY=/usr/lib/x86_64-linux-gnu/libpython3.7m.so \
-     -DPYTHON_INCLUDE_DIR=/usr/include/python3.7
+     -DPYTHON_EXECUTABLE=`which python3.8` \
+     -DPYTHON_LIBRARY=/usr/lib/x86_64-linux-gnu/libpython3.8m.so \
```

Comment:
as far as I know there is no `libpython3.8m.so` in this dir

CC @ilya-lavrenov @jiwaszki

Reply:
`m` is obsolete and used in 3.7 and older